static/app/views/dashboards/manage/dashboardTable.spec.tsx

```diff
@@ -344,7 +344,6 @@ describe('Dashboards - DashboardTable', function () {
       }
     );
 
-    expect((await screen.findAllByTestId('grid-head-cell')).length).toBe(5);
```

Comment:
instead of getting rid of this can we test to see if the prepend column is there? not sure if there's a testId for it or not

Reply:
the next line tests for this:
` expect(screen.getByLabelText('Favorite Column')).toBeInTheDocument();`

`Favorite Column` is from the icon in the header

Reply:
ooop i missed that one nvm